src/cache/factory.py

```diff
@@ -60,7 +60,7 @@ def get_cache() -> MaybeCache:
         object otherwise.
     """
     env = CacheEnvReader()
-    if is_cache_available() and env.cache_queries():
+    if is_cache_available():
```

Comment:
this change is already in #449, so perhaps we should merge #449 first and then double check what happens when rebasing, i.e. make sure nothing gets undone!